src/Database/Database.php

```
@@ -2768,25 +2913,43 @@ public function deleteAttribute(string $collection, string $id): bool
             }
         }
 
+        $collection->setAttribute('attributes', \array_values($attributes));
+        $collection->setAttribute('indexes', \array_values($indexes));
+
+        $shouldRollback = false;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$shouldRollback'. (undefined)

(UnusedLocalVariable)